service.subtitles.napiprojekt/resources/language/English/strings.po

```diff
@@ -0,0 +1,19 @@
+msgid ""
```

Comment:
you're using the old language folder layout. new:

resources/language/resource.language.en_gb/strings.po